### ISSUE TITLE
Best Model Parameters

### ISSUE THREAD

**Issue by pratyushmaini** [facebookresearch/BalancingGroups#3]
Do you plan to release the best models for all the algorithms and datasets? Based on the paper, it is not clear what were the best hyperparameter values since it provides mean and std over top 5. It would help make the work more accessible for researchers with lesser compute if you could also release the models! Thanks :)

**Comment by pratyushmaini** [comment]
Thanks. These numbers seem confusing:
1. Do you mean worst "group" test accuracy in the last column?
2. What is your definition of an epoch? It seems that you are using something non-standard. Does the gDRO model do that well in just 3 epochs? 
3. Are you checking validation at every iteration within an epoch? Why should the epochs be non integers otherwise.

**Comment by BadrYoubiIdrissi** [comment]
1. Yes we mean worst group accuracy. 
2. An epoch here is a full iteration on the training dataset. RWG does indeed do very well early on, as we explain in our paper the earlier we stop RWG the less it sees majority samples since it samples with replacement, therefore mimicking the behaviour of subsampling. 
3. No we are checking validation at the end of each epoch. The epochs are non integers because we are displaying the average best epoch over the different seeds.
4. We think it might be because the tuning is slightly overfitting on the validation data, since as you said the grid search is bigger. When we look up the best test worst group accuracies we get a few points more on most datasets.  That said, the previously reported numbers are well within our error bars.

**Comment by pratyushmaini** [comment]
Thank you!

**Comment by LishengSun** [comment]
Hello, could you please provide a more detailed explanation of how these top 5 hyperparameters were determined? As I understand from the paper, you ran 100 hyperparameter sets, each with 5 random seeds. Are these top 5 hyperparameters determined by averaging the results across the 5 random seeds? Thank you.
